src/sentry/lang/java/plugin.py

```diff
@@ -233,20 +231,22 @@ def process_frame(self, processable_frame, processing_task):
         new_frames = None
         raw_frames = None
         processing_errors = None
+        bare_frame = processable_frame.frame
+        key = frozenset(bare_frame.items())
```

Comment:
Could the frame be modified between where we generate the key from it? I'm assuming all modifications happen after this point, correct?

Reply:
yeah, and I believe this frame is actually immutable, we create two new lists of `raw_frames` (original) and `new_frames` (modified), but this stays untouched